src/backend/distributed/utils/log_utils.c

```diff
@@ -0,0 +1,25 @@
+/*-------------------------------------------------------------------------
+ *
+ * log_utils.c
+ *	  Utilities regarding logs
```

Comment:
It would kind of make sense to move ApplyLogRedaction over here, though it would complicate merge into enterprise.

Reply:
I guess we should move it now since this is the right file for it and at some point we will need to do that merge. (Assuming that we want methods to be in the right file)

Reply:
Also in community we still have that method as a "dummy" one. So I am updating the headers in this PR, I would only need to move the function in enterprise I guess.